src/util/address.rs

```diff
@@ -155,6 +153,19 @@ impl Address {
         }
     }
 
+    /// Create a pay to script address that embeds a witness pay to (uncompressed) public key
+    pub fn p2shwpkh_uncompressed (pk: &PublicKey, network: Network) -> Address {
```

Comment:
Use of uncompressed keys was discouraged before SegWit. We should not encourage doing this by adding a method to create scripts for them within SegWit.